ci/build.py

```diff
@@ -221,56 +221,62 @@ def script_name() -> str:
                         help="go in a shell inside the container",
                         action='store_true')
 
-    parser.add_argument("--docker-registry",
-                        help="Dockerhub registry name to retrieve cache from",
+    parser.add_argument("-d", "--docker-registry",
+                        help="Dockerhub registry name to retrieve cache from. Default is 'mxnetci'",
                         default='mxnetci',
                         type=str)
 
+    parser.add_argument("-c", "--cache", action="store_true",
```

Comment:
We already have that check in https://github.com/larroy/mxnet/blob/1aca0a1d3fe772f13a957175ddb42449c393f585/ci/build.py#L164. Just remove the default value in https://github.com/larroy/mxnet/blob/1aca0a1d3fe772f13a957175ddb42449c393f585/ci/build.py#L226 and we're good.

I don't see much reason to have two options to manage this functionality

Reply:
Please don't forget that we actually WANT the people to use the cache locally. If there's a problem, we should fix it.

Reply:
disagree. Locally we don't want to use cache, this is just an artifact for CI. please merge this PR.

Reply:
No, it is intended to be used locally as well.